pkg/polaris/graphql/infinity-k8s/infinity_k8s.go

```diff
@@ -229,6 +229,19 @@ func Wrap(client *polaris.Client) API {
 	return API{GQL: client.GQL, log: client.GQL.Log()}
 }
 
+// K8sObjectType is the type of the Kubernetes object. One between SLA,
+// INVENTORY, and SNAPSHOTS.
```

Comment:
Nit: K8sObjectType is the type of Kubernetes object that can be one of SLA, INVENTORY, or SNAPSHOTS.

Reply:
Will fix.